python/nammu/view/MenuView.py

```diff
@@ -107,15 +107,38 @@ def __init__(self, controller):
                       "Window": ["Display Model View"],
                       "Help": ["Help"]}
 
+        self.menu_positions = {}
         # Create menu items and add to menu bar
-        for menuName, keyEvent in menus.items():
+        for index, (menuName, keyEvent) in enumerate(menus.items()):
             menu = Menu(self,
                         menuName,
                         keyEvent,
                         menuItems[menuName],
                         separators[menuName])
             self.add(menu)
+            self.menu_positions[menuName] = index
 
     # Delegate methods not found here to view controller
     def __getattr__(self, name):
         return getattr(self.controller, name)
+
+    def getMenuByName(self, name):
```

Comment:
Try to not use camel case for variable names and methods to follow python convention. We should only use those for Java method calls.